pkg/cmd/builder/build.go

```diff
@@ -62,10 +62,10 @@ func Build(ctx context.Context, options *types.BuildCommandOptions, stdin io.Rea
 			return err
 		}
 	} else {
-		buildctlCmd.Stdout = stdout
+		buildctlCmd.Stdout = options.Stdout
```

Comment:
Should be set to `os.Stdout` if nil

Reply:
I think this is not related to this PR, I can make a split PR later